templates/101/evm/hardhat.config.ts

```diff
@@ -0,0 +1,26 @@
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+import "@typechain/hardhat";
+import "@nomicfoundation/hardhat-chai-matchers";
+import "@openzeppelin/hardhat-upgrades";
+
+const config: HardhatUserConfig = {
```

Comment:
Can you remind me if there is a field for verifyURL in non-zksync hardhat? How do we ensure folks using templates for EVM make use of the BE explorer verification endpoint and not EraVM endpoint?

Reply:
Can you confirm you can verify EVM contracts with contract-verifier endpoint? I don't think it handles it in an etherscan format yet? Our BE verification endpoint is etherscan format compatible. 

For clarity I am referencing this point: https://docs.zksync.io/zksync-era/unique-features/evm-interpreter/faq#how-do-i-verify-evm-contracts

Reply:
ah ok yes, it uses these endpoints, but it seems like the default hardhat verify plugin expects etherscan format. I still worked for me [here](https://sepolia.explorer.zksync.io/address/0x2BC1e2c4F356189Be5bD77Dc69e611592E205815#contract) though, so I'm guessing there would just be limitations?